ui/src/views/user/survey-detail/components/DetailResponses.vue

```diff
@@ -73,17 +55,23 @@ export default {
       }
     };
   },
+  computed: {
+    surveyId() {
+      return this.$route.params.id;
+    }
+  },
   created() {
-    this.survey = { ...this.survey, ...surveyDataSample };
+    this.getSurveyDatatableApi(this.surveyId);
   },
   methods: {
-    onClickDownloadButton() {
-      // do something
-      // fetch the api
-      this.$notify.toast("File has been downloaded");
+    getSurveyDatatableApi(surveyId) {
+      this.$notify.toast("Preparing data...");
+      userGetSurveyDatatable(surveyId).then(() => {
+        this.responsesUrl = "http://localhost:8000/dash_datatable/";
```

Comment:
Is this link static or dynamic? 
Also lease use VUE_APP_API_BASE_URL instead of localhost, unless there is no other choice XD

Reply:
it's static, basically we must call the api first so that BE prepares datatable for the selected surveyId (so the api has the 'dynamic' url), and then when we access the static link for the iframe it will be updated with datatable of the latest requested survey. i will change it to vue api base url XD

Reply:
Aha XD okie it will be intriguing to see if there are 2 users accessing the page at the same time, but its good enough for demo I guess. I have read the PR about this for more info.